client/admin/store/index.js

```diff
@@ -1,10 +1,13 @@
 import auth from '@/common/store/modules/auth';
+import { auth as authPlugin } from '@/common/store/plugins';
 import contentRepo from '@/admin/store/modules/content-repo';
-import plugins from '@/common/store/plugins';
 import programs from '@/admin/store/modules/programs';
 import Vue from 'vue';
 import Vuex from 'vuex';
 
+const MUTATION_TYPE = 'auth/login';
```

Comment:
This mutation is obsolete within admin store (login via learner (main) app). Also please rename the var so that is less generic.

Reply:
Right, I blindly applied updates without questioning it :upside_down_face: